command/ca/revoke.go

```diff
@@ -65,16 +65,29 @@ certificate lifetimes.
 **step ca revoke** currently only supports passive revocation. Active revocation
 is on our roadmap.
 
+A revocation request can be authorized using a JWK provisioner token, or using a
+client certificate.
+
+When you supply a serial number, you're prompted to choose a JWK provisioner,
```

Comment:
Currently, you're prompted to choose any one of your provisioners. But you should choose a JWK provisioner. Until we fix the code.